live: mention HA limitations in administration docs (#36752)
grafana/grafana: docs/sources/administration/set-up-for-high-availability.md

Currently alerting supports a limited form of high availability. [Alert notifications]({{< relref "../alerting/old-alerting/notifications.md" >}}) are deduplicated when running multiple servers. This means all alerts are executed on every server but alert notifications are only sent once per alert. Grafana does not support load distribution between servers.

## Grafana Live

> Grafana Live works with limitations in highly available setup, see details in [Grafana Live docs]({{< relref "../live/live-ha-setup.md" >}}).

Reviewer: ```suggestion
Grafana Live works with limitations in highly available setup. For redetails, refer to the [Grafana Live documentation]({{< relref "../live/live-ha-setup.md" >}}).
```

Author: Thanks, pushed a fix manually due to typo in suggestion.
